test/e2e/e2e.test.js

```diff
@@ -0,0 +1,172 @@
+const fs = require('fs')
+
+const landingPage = 'http://localhost:3001'
+const waitForTitle = title => page.waitForFunction(`document.title === '${title}'`, { timeout: 8000 })
+
+describe('Status page', () => {
+  beforeAll(async () => {
+    await page.goto(landingPage)
+    // get multiaddr of http api
+    const { apiHost, apiPort } = global.ipfs
+    const apiMultiaddr = `/ip4/${apiHost}/tcp/${apiPort}`
+    // point webui at real api
+    await page.evaluate((apiMultiaddr) => localStorage.setItem('ipfsApi', apiMultiaddr), apiMultiaddr)
+    // apply settings
+    await page.goto(landingPage)
```

Comment:
Are the settings instant-apply and you're just navigating back to start to reset for next test? Or is navigation change triggering the settings being saved?

Can you update comment to make clear which one is happening?

Reply:
`ipfsApi` is read from localStorage during webui "boot" done by [ipfs-redux-bundle](https://github.com/ipfs-shipyard/ipfs-redux-bundle).
I'll comment and move it to `test/e2e/setup/test-environment.js` so its done automatically before each test file.